data/002/0125/en_US.json

```diff
@@ -2,8 +2,8 @@
 	"002": {
 		"0125": {
 			"name": "Unknown",
-			"message": "",
-			"short_description": "Game id is invalid",
+			"message": "This device's access to online services\nhas been restricted by Nintendo.\n\nFor help, visit support.nintendo.com.",
+			"short_description": "Rom ID is invalid.",
 			"long_description": "Unknown cause.",
```

Comment:
Capitalise ROM, and ensure long description is filled out since we have a short description